src/backend/data/post.js

```diff
@@ -148,18 +148,26 @@ class Post {
       article.date = article.pubdate;
     }
 
+    if (Array.isArray(article.content) && determinePostType(article.link) === 'video') {
```

Comment:
When I did debugging all the of the Youtube post return `article.content` as an array off object instead of a string like what we expected which make the `html` content of all the related `video` post not being proccesed 

https://github.com/Seneca-CDOT/telescope/blob/4b2de81b153cd9607ce7c71ee5d36bee7bc5f8cd/src/backend/utils/html/index.js#L20-L21
 
![image](https://user-images.githubusercontent.com/43376325/151898713-d994ad4f-5153-445a-aa8e-11f6057a3c26.png)